tests/phpunit/test-class-wp-widget-image.php

```diff
@@ -0,0 +1,230 @@
+<?php
+
+class Test_WP_Widget_Image extends WP_UnitTestCase {
+
+	/**
+	 * @covers WP_Widget_Image::get_instance_schema
+	 */
+	function test_get_instance_schema() {
+		$WP_Widget_Image     = new ReflectionClass( 'WP_Widget_Image' );
+		$get_instance_schema = $WP_Widget_Image->getMethod( 'get_instance_schema' );
+		$get_instance_schema->setAccessible( true );
```

Comment:
Woah. You just made a protected method public!? I didn't know about this. http://php.net/manual/en/reflectionproperty.setaccessible.php

Looks like it isn't supported in PHP 5.2, so this will need to short-circuit with `$this->markTestSkipped()` if that is the PHP version.